Backpack-SwiftUI/Nudger/Classes/BPKNudger.swift

```diff
@@ -33,6 +33,7 @@ public struct BPKNudger: View {
     private var maxValue: Int
     private var step: Int
     private let minWidth: CGFloat = BPKSpacing.lg.value
+    @State var accessibilityPrefix: String?
```

Comment:
I'd make this private

Reply:
initially it was private, but after bringing the extension with the modifier, it seems, it's no longer possible to keep it private
unless you see another approach